### ISSUE TITLE
how you do  histogram matching

### ISSUE THREAD

**Issue by huangsusan** [JunMa11/HM_DataAug#2]
Hi：
In your code, I did not find where you apply histogram matching.Could you tell me how you do histogram matching?

**Comment by srivathsapv** [comment]
Yes, I have the same question. Can you please tell how you are doing histogram matching?

**Comment by huangsusan** [comment]
Hi:
The artical says how to use HM.You can check it.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "JunMa11/HM_DataAug"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年3月16日(星期二) 凌晨4:04
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [JunMa11/HM_DataAug] how you do  histogram matching (#2)





 
Yes, I have the same question. Can you please tell how you are doing histogram matching?

**Comment by yiia** [comment]
Sorry I am a beginner. How do I train from scratch? What is your command line during training? （python run_training.py .......???......）

**Comment by Isaaccheong95** [comment]
Hello, thank you for sharing your code, however, I am also unable to find the code for histogram matching, would you be able to upload that code as well?

**Comment by Brandy0k** [comment]
> Hello, thank you for sharing your code, however, I am also unable to find the code for histogram matching, would you be able to upload that code as well?

I have the same question.

**Comment by Rickustc** [comment]
I also have the same question...
